### ISSUE TITLE
Project status box on the landing page

### ISSUE THREAD

**Issue by Gregoor** [mozilla/voice-web#595]
Depends on #594

![project-status](https://user-images.githubusercontent.com/4051932/32338659-b79f378c-bff5-11e7-850c-97f96f81ec3e.jpg)

Requires BE changes (i.e. an API endpoint for getting the no of validated hours)

**Comment by caugner** [comment]
@Gregoor Has this been fixed with #621 or is there something left to do here?

**Comment by Gregoor** [comment]
good catch, @caugner. This is indeed already done!
